borgmatic/logger.py

```diff
@@ -42,7 +42,7 @@ def should_do_markup(no_color, configs):
         return False
 
     no_color_env = os.environ.get('NO_COLOR', None)
-    if no_color_env is not None and to_bool(no_color_env):
+    if no_color_env is not None:
```

Comment:
Super minor, but this could easily be on one line now (cutting out the intermediate variable).

Do not feel strongly.